CMSIS/NN/Include/arm_nnfunctions.h

```diff
@@ -1673,51 +1666,37 @@ extern    "C"
                                  q7_t * bufferA,
                                  q7_t * Im_out);
 
-  /**
-   * @brief s8 average pooling function
-   * @param[in]       dim_src_height     input tensor dimension
-   * @param[in]       dim_src_width      input tensor dimension
-   * @param[in]       dim_dst_height     output tensor dimension
-   * @param[in]       dim_dst_width      output tensor dimension
-   * @param[in]       stride_height      stride along y
-   * @param[in]       stride_width       stride along x
-   * @param[in]       dim_kernel_height  filter kernel size along y
-   * @param[in]       dim_kernel_width   filter kernel size along x
-   * @param[in]       padding_height     padding size along y
-   * @param[in]       padding_width      padding size along x
-   * @param[in]       act_min            Min clamping
-   * @param[in]       act_max            Max clamping
-   * @param[in]       ch_src             number of input tensor channels
-   * @param[in,out]   src                pointer to input tensor
-   * @param[in]       bufferA            Temporary buffer used for optimization. Use
-   *                                     arm_avgpool_s8_get_buffer_size() to get the size of required memory.
-   * @param[in,out]   dst                pointer to output tensor
-   * @return                             The function returns one of the following
-   *                                     <code>ARM_MATH_SIZE_MISMATCH</code> - Unsupported dimension of tensors
-   *                                     <code>ARM_MATH_SUCCESS</code> - Successful operation
-   *                                     <code>ARM_MATH_ARGUMENT_ERROR</code> - Implementation not available
+   /**
+   * @brief s8 average pooling function.
+   *
+   * @param[in, out] ctx            Function context (e.g. temporary buffer). Check the function
+   *                                definition file to see if an additional buffer is required.
+   *                                Optional function <API>_get_buffer_size() provides the buffer
+   *                                size if an additional buffer is required.
+   * @param[in]      pool_params    Pooling parameters
+   * @param[in]      input_dims     Input (activation) tensor dimensions. Format: [H, W, C_IN]
+   *                                Argument 'N' is not used.
```

Comment:
N is not used because batched pooling is not supported?

Reply:
@gmiodice Yes, batched pooling is not supported yet.